src/components/EditImageModal/index.jsx

```diff
@@ -120,8 +133,12 @@ export default class EditImageModal extends React.Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
+    if (this.state.isStatusAlertOpen) {
+      this.statusAlertRef.focus();
+    }
+
     if (this.state.pageNumber === 1 && prevState.pageNumber === 2) {
-      // TODO: add focus
+      this.dropZoneButtonRef.focus();
```

Comment:
I realized that I can't test the changes in here because of the disableLifecycleMethods :( If I misunderstood that and can actually test these things let me know!